model/model.go

```diff
@@ -225,10 +225,12 @@ func buildAppConfig(kubeClient *client.Client, service api.Service, routerConfig
 	if err != nil {
 		return nil, err
 	}
-	if routerConfig.Domain != "" {
-		for i, domain := range appConfig.Domains {
-			if !strings.Contains(domain, ".") {
+	for i, domain := range appConfig.Domains {
+		if !strings.Contains(domain, ".") {
+			if routerConfig.Domain != "" {
 				appConfig.Domains[i] = fmt.Sprintf("%s.%s", domain, routerConfig.Domain)
+			} else {
+				appConfig.Domains[i] = fmt.Sprintf("~^%s\\.(?<domain>.+)$", domain)
```

Comment:
I don't think it's a good idea to have put nginx-specific configuration in the model.  Yes... several model attributes are ostensibly named after nginx directives... and there's an open issue for fixing that, but in this case, we've inserted something nginx-specific as a _value_ in the model.  This will complicate the process of swapping nginx for something else in the future (which I think is a realistic possibility).